grails-app/controllers/org/bbop/apollo/AnnotationEditorController.groovy

```diff
@@ -903,8 +903,8 @@ class AnnotationEditorController extends AbstractApolloController implements Ann
             return
         }
         JSONObject featureContainer = createJSONFeatureContainer()
-        JSONObject sequenceObject = sequenceService.getSequenceForFeatures(inputObject)
-        featureContainer.getJSONArray(FeatureStringEnum.FEATURES.value).put(sequenceObject)
+        JSONArray featuresArray = requestHandlingService.getSequenceForFeatures(inputObject)
```

Comment:
@deepakunni3  what was the reasoning behind moving the method getSequenceForFeatures()  from SequenceService to RequestHandlingService?